### PR TITLE
schema: add `clean` to Makefile

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
LGTM

File: schema/Makefile [hunk 1]
```diff
@@ -16,3 +16,6 @@ validate: validate.go
 	go get -d ./...
 	go build ./validate.go
 
+clean:
```
Comment (wking) [inline schema/Makefile]: This is also `.PHONY`.

Comment (q384566678) [inline schema/Makefile]: I don't think that is necessary. `make clean` can be executed correctly.

Comment (wking) [inline schema/Makefile]: > I don't think that is necessary. `make clean1 can be executed correctly.

So can `make validate`, with or without the `.PHONY` declaration.  `.PHONY` *keeps* the target working even if someone creates a `clean` file.  And while that is unlikely, I don't see why we shouldn't protect ourselves from it.

Comment (q384566678) [inline schema/Makefile]: If you do not add `.PHONY`, `make validate` can not be repeated, the following error occurs：
`make: 'validate' is up to date.`
so add the `.PHONY`.

Comment (wking) [inline schema/Makefile]: > If you do not add `.PHONY`, `make validate` can not be repeated…

Fair.  An example closer to your `clean` would be the [the `clean` entry in the main `Makefile`][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/Makefile#L97-L98

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**